### ISSUE TITLE
UnsatisfiedLinkError

### ISSUE THREAD

**Issue by lekzeey** [bytedeco/javacv#470]
I get this error when I run OpenCVFaceRecognizer. Any clue what the problem is? Thanks

```
Exception in thread "main" java.lang.UnsatisfiedLinkError: no jniopencv_face in java.library.path
	at java.lang.ClassLoader.loadLibrary(ClassLoader.java:1867)
	at java.lang.Runtime.loadLibrary0(Runtime.java:870)
	at java.lang.System.loadLibrary(System.java:1122)
	at org.bytedeco.javacpp.Loader.loadLibrary(Loader.java:654)
	at org.bytedeco.javacpp.Loader.load(Loader.java:492)
	at org.bytedeco.javacpp.Loader.load(Loader.java:409)
	at org.bytedeco.javacpp.opencv_face.<clinit>(opencv_face.java:13)
	at OpenCVFaceRecognizer.main(OpenCVFaceRecognizer.java:84)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:144)
Caused by: java.lang.UnsatisfiedLinkError: /private/var/folders/bm/htqnz8316px3yw9m1wl4l2r80000gp/T/javacpp343774919637160/libjniopencv_face.dylib: dlopen(/private/var/folders/bm/htqnz8316px3yw9m1wl4l2r80000gp/T/javacpp343774919637160/libjniopencv_face.dylib, 1): Library not loaded: @rpath/libopencv_objdetect.3.1.dylib
  Referenced from: /private/var/folders/bm/htqnz8316px3yw9m1wl4l2r80000gp/T/javacpp343774919637160/./libopencv_face.3.1.dylib
  Reason: image not found
	at java.lang.ClassLoader$NativeLibrary.load(Native Method)
	at java.lang.ClassLoader.loadLibrary0(ClassLoader.java:1941)
	at java.lang.ClassLoader.loadLibrary(ClassLoader.java:1824)
	at java.lang.Runtime.load0(Runtime.java:809)
	at java.lang.System.load(System.java:1086)
	at org.bytedeco.javacpp.Loader.loadLibrary(Loader.java:637)
	... 9 more

```

**Comment by lekzeey** [comment]
Fix : I used an opencv 2.4 and modified the opencvfacerecognizer

**Comment by saudet** [comment]
Looks like `opencv_face` depends on `opencv_objdetect`. I've fixed that in the latest commit (link above), but for now we can call the following to work around:
```java
Loader.load(opencv_objdetect.class);
```
Thanks for reporting!
